src/components/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/calculator',
+      text: 'Calculator',
+    },
+    {
+      id: 3,
+      path: '/quote',
+      text: 'Quote',
+    },
+  ];
+
+  return (
+    <nav className="navBar">
+      <ul>
+        {links.map((link) => (
+          <li key={link.id}>
+            <NavLink to={link.path}>{link.text}</NavLink>
```

Comment:
[OPTIONAL]
- Wonderful display of the navigation links. They're working as expected. 👏 
What do you think about adding a `className` to the links and styling them to make them look better and more visible?

I played around with a bit of styling, and for ease, you can use it as well(if you like it) 😀 

```
.nav-link {
  text-decoration: none;
  color: #fff;
  transition: 300ms;
}

.nav-link:hover {
  color: #f4913e;
}
```
_Assuming the className given is `nav-link`_